util/merkle/src/snapshot/mod.rs

```diff
@@ -91,6 +90,12 @@ impl<'a> Restore<'a> {
     }
 }
 
+impl std::fmt::Debug for Restore {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
+        write!(f, "Restore {{ pending: {:?} }}", self.pending)
```

Comment:
You can use this: https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html
```rust
        fmt.debug_struct("Restore")
           .field("pending", &self.pending)
           .field("unresolved", "<...>")
           .finish()
```

Reply:
Fixed.